### PR TITLE
Disable RSpec/MultipleMemoizedHelpers

### PR DESCRIPTION
**Pull request by rejeep**
The description of the cop is:

> Checks if example groups contain too many `let` and `subject` calls.

I don't understand why that would be prohibited by default since it seems to me to be a good thing to have let blocks that can be extended in contexts.

### CODE REVIEW

**Review by JacobJNilsson — APPROVED** [review]
👍

**Review by JacobJNilsson — APPROVED** [review]

**Comment by JacobJNilsson** [comment]
I'm guessing I don't have write access and can't approve these changes. But did I not approve a PR before?

**Comment by rejeep** [comment]
I added access to BIC

**Pull request merged**
